src/ontology/uberon.Makefile

```diff
@@ -121,6 +126,11 @@ $(TMPDIR)/plugins/sssom.jar:
 	mkdir -p $(TMPDIR)/plugins
 	curl -L -o $@ https://github.com/gouttegd/sssom-java/releases/download/sssom-java-0.6.1/sssom-robot-plugin-0.6.1.jar
 
+# Ditto for the specific Uberon plugin
+$(TMPDIR)/plugins/uberon.jar:
+	mkdir -p $(TMPDIR)/plugins
+	curl -L -o $@ https://github.com/gouttegd/uberon-robot-plugin/releases/download/uberon-robot-plugin-0.1.0/uberon.jar
```

Comment:
Is there a _really_ good reason for splitting all the gouttegd-robot-extensions into multiple jars?

Reply:
Yes. The SSSOM plugin is a generic plugin to use SSSOM files in a ROBOT pipeline. There’s nothing specific to Uberon in it.

By constrast, the Uberon plugin, as its name implies, is _very_ specific to Uberon. I doubt any other project than Uberon will ever need a `merge-species` command.

I invert the question: give me a _really_ good reason why those plugins should be merged into a single one?

Reply:
You must be joking… Building Uberon requires a ~2.5 GB Docker image, and you’re concerned about the size of two Jar files of ~2,600 KB (the SSSOM plugin) and ~23 KB (the Uberon plugin)?

In any case, merging the two plugins would not change anything about their size, because each plugin only contains its own code and nothing else – all the common code, such as the OWL API, is provided by ROBOT. So yeah, we could merge the two plugins, and then instead of having two files of 2,600 KB and 23 KB, we would have only one of 2,623 KB…